### DISCUSSION TITLE
Certificados da plataforma

### DISCUSSION THREAD

**Discussion by i-Lucas** [balta-io/forum#46] [Tira Dúvidas] [answered]
Olá bom dia, tudo bem ? 

Gostaria de saber se os **cursos gratuitos da plataforma emitem certificação** ao término do mesmo.

E se esses certificados são reconhecidos por alguma instituição reguladora.

Estou pensando em virar membro premium da plataforma, mas primeiro preciso saber essas informações.

**Comment by andrebaltieri** [comment, marked as answer]
Boa tarde, @i-Lucas, como vai?

Temos certificados sim (Incluindo cursos gratuitos), porém não há uma instituição reguladora para os mesmos.
